metricbeat/module/kubernetes/apiserver/apiserver.go

```diff
@@ -25,16 +25,52 @@ import (
 func init() {
 	mapping := &prometheus.MetricsMapping{
 		Metrics: map[string]prometheus.MetricMap{
-			"apiserver_request_count":     prometheus.Metric("request.count"),
+
+			// Process metrics
+			"process_cpu_seconds_total":     prometheus.Metric("process.cpu.sec"),
+			"process_resident_memory_bytes": prometheus.Metric("process.memory.resident.bytes"),
+			"process_virtual_memory_bytes":  prometheus.Metric("process.memory.virtual.bytes"),
+			"process_open_fds":              prometheus.Metric("process.fds.open.count"),
+			"process_start_time_seconds":    prometheus.Metric("process.started.sec"),
+
+			// HTTP server metrics
+			"http_request_duration_microseconds": prometheus.Metric("http.request.duration.us"),
+			"http_request_size_bytes":            prometheus.Metric("http.request.size.bytes"),
+			"http_response_size_bytes":           prometheus.Metric("http.response.size.bytes"),
+			"http_requests_total":                prometheus.Metric("http.request.count"),
+
+			// REST metrics
+			"rest_client_requests_total": prometheus.Metric("client.request.count"),
+
+			// Deprecated, remove in future releases
 			"apiserver_request_latencies": prometheus.Metric("request.latency"),
+
+			"apiserver_request_total":                 prometheus.Metric("request.count"),
+			"apiserver_current_inflight_requests":     prometheus.Metric("request.current.count"),
+			"apiserver_longrunning_gauge":             prometheus.Metric("request.longrunning.count"),
+			"etcd_object_counts":                      prometheus.Metric("etcd.object.count"),
+			"apiserver_audit_event_total":             prometheus.Metric("audit.event.count"),
+			"apiserver_audit_requests_rejected_total": prometheus.Metric("audit.rejected.count"),
 		},
 
 		Labels: map[string]prometheus.LabelMap{
-			"client":      prometheus.KeyLabel("request.client"),
-			"resource":    prometheus.KeyLabel("request.resource"),
-			"scope":       prometheus.KeyLabel("request.scope"),
-			"subresource": prometheus.KeyLabel("request.subresource"),
-			"verb":        prometheus.KeyLabel("request.verb"),
+			"client":      prometheus.KeyLabel("client"),
+			"code":        prometheus.KeyLabel("code"),
+			"contentType": prometheus.KeyLabel("content_type"),
+			"dry_run":     prometheus.KeyLabel("dry_run"),
+			"requestKind": prometheus.KeyLabel("kind"),
+
+			"verb":        prometheus.KeyLabel("verb"),
+			"scope":       prometheus.KeyLabel("scope"),
+			"resource":    prometheus.KeyLabel("resource"),
+			"subresource": prometheus.KeyLabel("subresource"),
+			"component":   prometheus.KeyLabel("component"),
+			"group":       prometheus.KeyLabel("group"),
+			"version":     prometheus.KeyLabel("version"),
+
```

Comment:
Labels here seem to be separated in three blocks, it may be useful to add some comments with a brief title for each block.